packages/trader/src/Modules/CFD/Containers/cfd-password-manager-modal.tsx

```diff
@@ -261,15 +277,15 @@ const CFDPasswordManagerTabContent = ({
     platform,
     onChangeActiveTabIndex,
     account_group,
-}) => {
-    const [active_tab_index, setActiveTabIndex] = React.useState(0);
-    const [error_message_investor, setErrorMessageInvestor] = React.useState('');
-    const [is_submit_success_investor, setSubmitSuccessInvestor] = React.useState(false);
+}: TCFDPasswordManagerTabContent) => {
+    const [active_tab_index, setActiveTabIndex] = React.useState<number>(0);
+    const [error_message_investor, setErrorMessageInvestor] = React.useState<string>('');
+    const [is_submit_success_investor, setSubmitSuccessInvestor] = React.useState<boolean>(false);
 
     // view height - margin top and bottom of modal - modal title - modal content margin top and bottom - table title
-    const password_container_height = 'calc(100vh - 84px - 5.6rem - 8.8rem - 4rem)';
-    const validatePassword = values => {
-        const errors = {};
+    const container_height = 'calc(100vh - 84px - 5.6rem - 8.8rem - 4rem)';
+    const validatePassword = (values: { old_password: string; new_password: string; password_type: string }) => {
+        const errors: { new_password?: string; old_password?: string } = {};
```

Comment:
my suggestion:
```suggestion
        const errors: FormikErrors<TFormValues> = {};
```
* please see my previous comment about TFormValues 🙂